VideoLocker/src/main/java/org/edx/mobile/player/Player.java

```diff
@@ -331,15 +330,25 @@ public void surfaceCreated(SurfaceHolder holder) {
                         if (!isPlaying()) {
                             seekTo(lastCurrentPosition);
                         }
-                    } catch(Exception ex) {
-                        logger.error(ex);;
+                    } catch (Exception ex) {
+                        logger.error(ex);
+                        ;
                     }
                 }
 
                 @Override
-                public void surfaceChanged(SurfaceHolder holder, int format,
-                        int width, int height) {
-                    // nothing to be done here
+                public void onSurfaceTextureSizeChanged(SurfaceTexture surface, int width, int height) {
+
+                }
+
+                @Override
+                public boolean onSurfaceTextureDestroyed(SurfaceTexture surface) {
+                    return false;
```

Comment:
The documentation says "Most applications should return true" for this. Is this what we want?